media-video/aravis/aravis-0.7.2.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit meson xdg
+
+DESCRIPTION="Library for video acquisition using Genicam cameras"
+HOMEPAGE="https://github.com/AravisProject/aravis"
+
+if [[ ${PV} = 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/AravisProject/${PN}"
+else
+	MY_P="${PN^^}_${PV//./_}"
+	SRC_URI="https://github.com/AravisProject/${PN}/archive/${MY_P}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="LGPL-2+"
+SLOT="0"
+# FIXME: As of right now tests are always built, once that changes a USE flag
+# should be added. c.f. https://github.com/AravisProject/aravis/issues/286
+IUSE="gtk-doc fast-heartbeat gstreamer introspection packet-socket usb viewer"
+
+GST_DEPEND="
+	media-libs/gstreamer:1.0
+	media-libs/gst-plugins-base:1.0
+"
+BDEPEND="
+	dev-util/glib-utils
+	virtual/pkgconfig
+	gtk-doc? (
+		dev-util/gtk-doc
+		app-text/docbook-xml-dtd:4.3
+	)
+	introspection? ( dev-libs/gobject-introspection:= )
+"
+DEPEND="
+	dev-libs/glib
```

Comment:
Why drop slot from glib? Its there to help out portages slow work and to prevent possible untested breakages in future.

Rest looks good.

Reply:
There's only slot 2 in the tree, what's the point of specifying the slot then?

Reply:
As I said, it's to help out portage and prevent possible incoming breakage when, say, `:3` gets added.

Reply:
Reintroduced :)